src/types/onyx/PersonalDetails.ts

```diff
@@ -76,6 +76,8 @@ type PersonalDetails = {
     payPalMeAddress?: string;
 };
 
+type PersonalDetailsList = Record<string, PersonalDetails>;
```

Comment:
Since you added this new type, could you check for `Record<string, PersonalDetails>` usage in the repo and make them use this new one? Thanks!